app/models/miq_report_result/result_set_operations.rb

```diff
@@ -41,10 +41,10 @@ def result_set_for_reporting(report_result, options)
       if result_set.present? && report
         if filtering_enabled?(options)
           result_set, count_of_full_result_set = report.filter_result_set(result_set, filter_options(options))
-          allowed_columns_to_format = report.col_order - filter_options(options).keys
+          allowed_columns_to_format = report.cols - filter_options(options).keys
```

Comment:
no need to format columns that are not displayed.
I like it how it was before

Reply:
we need to display also columns from  `#cols` - at least for chargeback and I believe that columns it is valid all reports

so I am putting here both(cols and col_order) - let me know is this unnecessary

Reply:
did you revert this?
`col_order` has some columns that are not in `cols`. And we are only displaying `col_order` via formatting that I'm aware.